VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -396,13 +396,13 @@ def get_total_used_size(self):
                             total_used = total_used + int(used)
                             self.logger.log("Adding only root device to size calculation. Device name : {0} used space in KB : {1} mount point : {2} fstype : {3}".format(device,used,mountpoint,fstype),True)
                             self.logger.log("Total Used Space: {0}".format(total_used),True)
-                    #Handling a case where LunList is empty for UnmanagedVM's
-                    elif(self.LunListEmpty == True and device != resource_disk_device):
+                    #Handling a case where LunList is empty for UnmanagedVM's and failures if occurred( as we will billing for all the non resource disks)
+                    elif( (self.size_calc_failed == True or self.LunListEmpty == True) and device != resource_disk_device):
```

Comment:
We can convert this into a helper function that returns if AllDisksToBeIncluded.